tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesView.xaml

```diff
@@ -10,6 +10,11 @@
                 Visibility="{x:Bind ViewModel.IsFeaturePresent, Mode=OneWay}">
         <ctControls:SettingsExpander x:Uid="QuietBackgroundProcesses" IsExpanded="False" Margin="{ThemeResource SettingsCardMargin}">
 
+            <!-- Description / error message slot -->
```

Comment:
Tiny nit: I'd do the header icon section above the Description section, just so the code is addressing the parts of the SettingsExpander from left to right. But that's so tiny.